_includes/specials/bech32/16-checklist.md

```diff
@@ -0,0 +1,54 @@
+Recently, an Optech contributor surveyed many popular wallets and Bitcoin
```

Comment:
I think you need to update 2019-03-19-bech32-sending-support.md to include this.